### PR TITLE
LIMITED_RETRY step result

### PR DESCRIPTION
**Pull request by radix**
This introduces a new StepResult.LIMITED_RETRY`, and the planner returns it for servers in `Destiny.WAIT`.

Nothing else is currently done with it.

### CODE REVIEW

**Comment by radix** [comment]
jenkins please run integration tests

**Comment by radix** [comment]
jenkins please run integration tests

File: otter/convergence/service.py [hunk 1]
```diff
@@ -309,8 +309,8 @@ def convergence_succeeded(scaling_group, group_state, servers, now):
     yield Effect(
         UpdateServersCache(
             scaling_group.tenant_id, scaling_group.uuid, now,
-            [merge(thaw(s.json), {"_is_as_active": True})
-                for s in servers if s.state != ServerState.DELETED]))
+            [thaw(s.json.set("_is_as_active", True))
```
Comment (radix) [inline otter/convergence/service.py]: Sorry, this is just some really minor bikeshedding.

Comment (manishtomar) [inline otter/convergence/service.py]: No problem :). Its actually better.

**Comment by manishtomar** [comment]
LGTM :)

**Pull request merged**